repo.lb

```diff
@@ -15,11 +15,20 @@ import os
 import sys
 import glob
 
-rootpath = localpath("ext", "modm-devices", "tools", "device")
+rootpath = localpath("ext/modm-devices/tools/device")
 sys.path.append(rootpath)
 
-import modm.pkg
-import modm.parser
+try:
+    import modm.pkg
+    import modm.parser
+except Exception as e:
+    print(e, "\n")
+    print("You might be missing the git submodules in `ext/`.\n"
+          "Try:\n"
+          "    cd modm\n"
+          "    git submodule update --init\n"
+          "then build again.")
+    exit(1)
```

Comment:
This should solve the "Oh, I've not added the `--recursive` flag to my `git clone` command", that happens fairly often.